app/controllers/reasons/ChangeUnderpaymentReasonController.scala

```diff
@@ -97,16 +97,24 @@ class ChangeUnderpaymentReasonController @Inject()(identify: IdentifierAction,
           items = Seq(
             ActionItemHelper.createChangeActionItem(
               controllers.reasons.routes.ChangeUnderpaymentReasonDetailsController.onLoad(underpaymentReason.boxNumber).url,
-              messages("changeUnderpaymentReason.values.change")
+              messages("changeUnderpaymentReason.values.original.change")
             )
-          ),
-          classes = "govuk-!-padding-bottom-1")
-        ),
-        classes = "govuk-summary-list__row--no-border"
+          )
+        )
+        )
       ),
       SummaryListRow(
         key = Key(content = Text(messages("changeUnderpaymentReason.amended")), classes = "govuk-!-width-two-thirds govuk-!-padding-top-0"),
-        value = Value(content = HtmlContent(underpaymentReason.amended), classes = "govuk-!-padding-top-0")
+        value = Value(content = HtmlContent(underpaymentReason.amended), classes = "govuk-!-padding-top-0"),
+        actions = Some(Actions(
+          items = Seq(
+            ActionItemHelper.createChangeActionItem(
+              controllers.reasons.routes.ChangeUnderpaymentReasonDetailsController.onLoad(underpaymentReason.boxNumber).url,
+              messages("changeUnderpaymentReason.values.amended.change")
+            )
+          ),
+          classes = "govuk-!-padding-bottom-1")
```

Comment:
The padding classes need to be consistent across all 3 elements (key, value, actions) on every Summary List Row. I the majority of these cases there will no longer be any padding classes uses (on the width ones).